### ISSUE TITLE
Feature Request: Option for larger transparency control slider

### ISSUE THREAD

**Issue by john-99** [osmandapp/Osmand#4841]
When showing an overlay map as well as the main map, the slider to control the transparency of the overlay is so small that it can take many attempts to position a finger accurately enough to operate it. Would it be possible to add a setting to choose a finger-friendly size for this slider?

I'm using osmand+ 2.8.2 on a Galaxy tablet. My use case is to display a vector map of my own creation as the main map and to be able to cross-fade between that and the regular osmand tiles map. My vector map is a representation of a historical map of the area so the slider provides a way of cross-fading between "then" and "now".

**Comment by iriman** [comment]
what about a hidden slider that starts from profile button and ends to menu/dashboard button? Slider operation starts by touching any of these two buttons and then dragging.
or if starts off from right side to left or buttom.
![picsart_12-09-11 10 50](https://user-images.githubusercontent.com/8467903/33793721-dcd6e7f4-dcd1-11e7-8561-4542d14531eb.jpg)

**Comment by john-99** [comment]
I think I see what you mean, but I'm afraid I'm not convinced. I'm really not happy about hidden interface features in general, and if the slider is hidden how do you pick it up again if you've previously moved it to the half-way point to show a mixture of the maps? I understand that screen real estate is precious but I'd prefer to lose some if it makes for a usable control whose setting is visible. To be able to set a "Big Slider" user preference would be ideal, I think.

**Comment by iriman** [comment]
yes, half-way point is an issue I hadn't think about but may be possible to catch the handle on every trigger and work relatively.
I agree, this may be more complicated rather  applicable for the users. A visible slider is more straight forward to work with.
Anyway, developers may know a better solution.
Just some suggestions.
